source/api_documentation/network_integration/advertiser_campaigns/_get_advertiser_campaign_unarchive.rst

```diff
@@ -0,0 +1,15 @@
+.. container:: endpoint-long-description
```

Comment:
@mikeweaver 
I looked through other parts of the NI API. 
Advertiser Campaigns seems to be the only place where GET requests do more than read.
I'll remove the GET **unarchive** example from this ticket and rip out the existing GET **archive** example.